tiledb/api/src/config.rs

```diff
@@ -265,6 +286,24 @@ impl<'cfg> Iterator for ConfigIterator<'cfg> {
     }
 }
 
+/// Convenience for setting some of the more commonly-used configuration options.
+#[derive(Clone, Debug)]
+pub enum CommonOption {
+    /// Sets an AES256GCM encryption key.
+    Aes256GcmEncryptionKey(Vec<u8>),
```

Comment:
I wonder if this shouldn't be defined as something like:

```rust
pub enum CommonOption {
    Encryption(array::Encryption, Option<String>)
}
```

This is only a vague "might be better, might not" suggestion. So feel free to do what you want with it.

Reply:
My thinking here (and the way I am following up in the tables repo so far) is that other encryption types might have more or different arguments.  So for now I'm just going with this.